### DISCUSSION TITLE
User testing of the StreetComplete

### DISCUSSION THREAD

**Discussion by DwynDwe** [streetcomplete/StreetComplete#5974] [Feedback]
Hi! I did user testing as part of my diploma thesis. Since the feedback is a few pages long, I am attaching it as a PDF file to make it easier to read. I hope that some of my findings will be useful. If you have any questions, please do not hesitate to ask. I had to shorten the results so I wouldn't bore you :) Thanks for developing such a great app!

If you want to reach me regarding [my diploma thesis](https://is.muni.cz/th/wjetm/?lang=en;vysl=21710;id=336344) or some particular results, please email 241061@muni.cz.
[StreetComplete_user_testing.pdf](https://github.com/user-attachments/files/17447313/StreetComplete_user_testing.pdf)

**Comment by matkoniecz** [comment]
I made a cursory look and I see mention of problem with a translation - are you interested in providing a fix? See https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app

**Comment by DwynDwe** [comment]
I have corrected the problematic phrase. In the next update, I will check if it is translated everywhere.

**Comment by matkoniecz** [comment]
Looking more at the text - one easily actionable idea is hiding width quest by default. I guess that issue for that can be created?

@westnordost - what is you opinion on potentially doing this? On one side it may make this quest to hidden, on the other hand they are both confusing and time-consuming, especially for newbies

![screen](https://github.com/user-attachments/assets/fdccd576-109f-47f0-b4b0-69da88257e26)

-----------

(see sentence before points and last point)

![screen01](https://github.com/user-attachments/assets/6a5fc2e3-22bc-4465-94a5-142698a2bb4d)

**Comment by matkoniecz** [comment]
@DwynDwe You mention "a poorly described task to find out the height of the curb" - is it also a translation issue? Or is it unclear also in English?

**Comment by westnordost** [comment]
An interesting read! Also, the findings can be somewhat useful to improve the app's usability, although, many things that had been brought up by the test users are known issues for which there is no easy solution.

> I liked the base map

Did you test with Dark mode or light mode?

> "During the next 10 minutes, map as many tasks as possible"

(By the way) it is not the primary design goal of StreetComplete to optimize for that. The focus is on convenience and ease of use rather than on efficiency. By asking users to to be as efficient as possible with a time limit of 10 minutes with an app that is not made to be efficient (as primary goal), you may have gotten some slightly skewed results. E.g. quests that take longer (opening hours, road width, ...) or require some getting to know it (building types, possibly any road-side-quest) will be evaluated worse for the simple reason that they take longer to solve (at first).

> More difficult tasks should have [...] an accessible help that could be opened at any time during the completion of the task (and would
contain a detailed description of the problem).

This was implemented in v58. Any quest can have some hint text + image(s) now.

**Comment by DwynDwe** [comment]
@matkoniecz  I looked in the app and the curb task is the same in English and Czech (in the way of describing what the user should do). I also looked into my notes from the testing and the participant didn't know that he should look for a curb on the sidewalk's edge (transition from sidewalk to road). So he tried to find the correct one by approximating a position by GPS location and he found one on the edge of the sidewalk and grass island. Maybe more descriptive text of the task would help.

@westnordost  We tested only light mode. The question for the base map was there because Czech users are used to detailed base/topographic maps (like [mapy.cz](https://en.mapy.cz/turisticka?x=16.5775983&y=49.1942021&z=11)).

I know that the main goal of StreetComplete is not to map as fast as possible. However, it was necessary to limit the testing time and prevent some participants from getting stuck on one task. I admit that this may have affected the skipping of some tasks. But opening hours, for example, was a very frequently solved task. Even so, I think that participants skipped most of the tasks because they found them difficult or unattractive (often accompanied by the comment "I don't want to do that").

**Comment by mnalis** [comment]
- (ref. https://github.com/streetcomplete/StreetComplete/issues/6256)
